transform/snowflake-dbt/models/mattermost/hourly/server_fact.sql

```diff
@@ -31,9 +31,7 @@ WITH sdd AS (
             FROM {{ ref('server_daily_details') }} SDD
             WHERE timestamp <= CURRENT_TIMESTAMP
             {% if is_incremental() %}
-            AND ((CASE WHEN in_security OR in_mm2_server THEN timestamp ELSE NULL END) > (SELECT MAX(last_active_date) - INTERVAL '2 HOURS' FROM {{this}})
-            OR NOT EXISTS (select 1 FROM {{this}} SF WHERE SF.SERVER_ID = SDD.SERVER_ID)
-            )
+            AND (CASE WHEN in_security OR in_mm2_server THEN timestamp ELSE NULL END) > (SELECT MAX(last_active_date) - INTERVAL '2 HOURS' FROM {{this}})
```

Comment:
```suggestion
            HAVING MAX(CASE WHEN in_security OR in_mm2_server THEN timestamp ELSE NULL END) > (SELECT MAX(last_active_date) - INTERVAL '2 HOURS' FROM {{this}})
```

Reply:
^this is what the query used to be

Reply:
you are correct about that, I'm testing the impact of adding the other condition to add the missing servers here.